provider-attributes.json

```diff
@@ -0,0 +1,676 @@
+{
+  "host": {
+    "key": "host",
+    "type": "string",
+    "required": true,
+    "description": "Name of the host. Ex.: 'd3akash.cloud'."
+  },
+  "email": {
+    "key": "email",
+    "type": "string",
+    "required": true,
+    "description": "Email associated with the provider."
+  },
+  "organization": {
+    "key": "organization",
+    "type": "string",
+    "required": true,
+    "description": "Name of the organization that owns the provider."
+  },
+  "provider-website": {
```

Comment:
Is the `provider-` prefix necessary for the `website` and `status-page`? Why do we need a prefix here, but not on `email` or `organization`?